scripts/zones/Lower_Delkfutts_Tower/Zone.lua

```diff
@@ -54,19 +78,36 @@ function onRegionLeave(player, region)
 end
 
 function onEventUpdate(player, csid, option)
+    if
+        csid == 51 and
+        option == 1
+    then
+        local metPrishe = 0
+
+        if player:hasCompletedMission(COP, tpz.mission.id.cop.DISTANT_BELIEFS) then
+            metPrishe = 1
+        end
+        player:updateEvent(0, metPrishe, 0, 0, 0, 0)
+    end
 end
 
 function onEventFinish(player, csid, option)
     if csid == 15 then
         player:setCharVar("ZilartStatus", 2)
-    elseif csid == 4 and option == 1 then
+    elseif
+        csid == 4 and
+        option == 1 then
         if player:getCharVar("option") == 1 then
             player:setPos(-28, -48, 80, 111, 157)
         else
             player:setPos(-51, -48, -40, 246, 157)
         end
         player:setCharVar("option", 0)
-    elseif csid == 4 and (option == 0 or option >= 3) then
+    elseif
+        csid == 4 and
+        (option == 0 or
```

Comment:
so uh... I guess ya just did that to all the if/thens? [we normally only break the wide ones](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#general-code-guidlines-all-languages)